src/sentry/ingest/consumer/processors.py

```diff
@@ -280,13 +280,17 @@ def process_individual_attachment(message: IngestMessage, project: Project) -> N
         group_id = event.group_id
 
     attachment = message["attachment"]
-    attachment = attachment_cache.get_from_chunks(
-        key=cache_key, type=attachment.pop("attachment_type"), **attachment
-    )
-    if attachment.type not in ("event.attachment", "event.view_hierarchy"):
-        logger.error("invalid individual attachment type: %s", attachment.type)
+    attachment_type = attachment.pop("attachment_type")
+    if attachment_type not in ("event.attachment", "event.view_hierarchy"):
+        logger.error("invalid individual attachment type: %s", attachment_type)
+
+        # clear invalid attachments from the cache directly
+        attachment.delete()
```

Comment:
The comment says `clear invalid attachments from the cache directly`, but `attachment` is an entry in a mapping (`message`) here. Can you clarify what cache is being referenced here?

Reply:
indeed, the variable was wrongly shadowed. I moved the code around and gave each its own unique name.